metrics/deploy/deployment.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     app.kubernetes.io/name: ocs-metrics-exporter
     app.kubernetes.io/version: 0.0.1
   name: ocs-metrics-exporter
-  namespace: openshift-storage
+  namespace: rook-ceph
```

Comment:
this will be reverted back isnt it?

Reply:
Yes. These files are just for testing.
Will revert before merging.